orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/StartWaitingExecutionsHandler.kt

```diff
@@ -61,12 +65,21 @@ class StartWaitingExecutionsHandler(
       // when not purging the queue, run the messages in the order they came in
       pendingExecutionService.popOldest(message.pipelineConfigId)
     }
-      ?.let {
-        // spoiler, it always is!
-        if (it is ExecutionLevel) {
-          log.info("Starting queued pipeline {} {} {}", it.application, it.executionId)
-        }
-        queue.push(it)
+
+    if (pendingMessage != null) {
+
+      // spoiler, it always is!
+      if (pendingMessage is ExecutionLevel) {
+        log.info("Starting queued pipeline {} {} {}", pendingMessage.application, pendingMessage.executionId)
       }
+      queue.push(pendingMessage)
+    } else {
+      interlink?.publish(
+        StartPendingInterlinkEvent(
+          ExecutionType.PIPELINE,
+          "NO_EXECUTION_ID",
```

Comment:
nit: we might want to make it a symbolic constant in an interface somewhere, maybe in `InterlinkEvent`?

Reply:
i think i can actually eliminate, let me see